js/graph_spectrum_plot.js

```diff
@@ -689,3 +822,4 @@ GraphSpectrumPlot._invalidateCache = function() {
 GraphSpectrumPlot._invalidateDataCache = function() {
     this._cachedDataCanvas = null;
 };
+
```

Comment:
:space_invader:

Reply:
We don't want an empty line at the end? We have some rule about it?

Reply:
I don't think we have a rule. Is it having a line break at the end of the file that you are thinking about?

Reply:
True, is the only change I've done here, so I suppose you are asking to remove it 😉

Reply:
Sorry, from the phone I didn't notice there were TWO empty lines at the end. I have removed one. :-)